test/Generics/validate_stdlib_generic_signatures.swift

```diff
@@ -0,0 +1,6 @@
+// Verifies that all of the generic signatures in the standard library are
+// minimal and canonical.
+
+// RUN: %target-typecheck-verify-swift -typecheck %s -verify-generic-signatures Swift
```

Comment:
So this passes now? :)

Reply:
YES!

Reply:
My comfort level in GSB minimization for ABI stability has increased 100x